setup.py

```diff
@@ -41,9 +41,9 @@
     platforms="Posix; MacOS X; Windows",
     install_requires=[
         "attrs~=20.2",
+        "cattrs~=1.8",
```

Comment:
Looks like this commit (`src(): update dependencies`) should be first in the PR?

See also 8.2.2 of the SDLC for how dependency updates need to be done: each one should be a separate commit like `deps(cattrs): add v1.8` and `deps(munch): remove`.

Reply:
:+1: